### PR TITLE
chore: remove `--silent` flag

### PR DESCRIPTION
**Pull request by KubaJastrz**
I didn't like that I couldn't use `console.log` to debug some values in tests, so I mocked `console.warn` globally.

Not sure if this is the best solution but it works 🤷‍♂

### CODE REVIEW

File: test/integration/createTippy.test.js [hunk 1]
```diff
@@ -179,8 +179,6 @@ describe('instance.destroy()', () => {
     jest.advanceTimersByTime(500);
 
     expect(spy).not.toHaveBeenCalled();
-
-    spy.mockRestore();
```
Comment (atomiks) [inline test/integration/createTippy.test.js]: Why was this removed?

File: test/setup.js [hunk 2]
```diff
@@ -18,3 +18,13 @@ global.requestAnimationFrame = cb => cb();
 
 // Only relevant for bindGlobalEventListeners.test.js
 Object.defineProperty(window.navigator, 'platform', {value: 'iPhone'});
+
+// Prevents console from spamming test output while still allowing for debugging
+// while writting tests
```
Comment (atomiks) [inline test/setup.js]: `writing` spelling mistake

File: test/integration/createTippy.test.js (hunk 1 above)
Comment (KubaJastrz) [reply test/integration/createTippy.test.js]: This spy is not used anymore during this test, so it's redundant.

Comment (atomiks) [reply test/integration/createTippy.test.js]: What if I add another spy later on in the file?

Comment (KubaJastrz) [reply test/integration/createTippy.test.js]: I meant this particular test case. Other `it`s will work fine.

Cleaning up is usually good even if not needed, but I haven't seen manual restoring in other test cases 🙄

Comment (atomiks) [reply test/integration/createTippy.test.js]: tbh I don't understand properly, but I'll trust you're right..

> I haven't seen manual restoring in other test cases

Should that be another PR?

Comment (KubaJastrz) [reply test/integration/createTippy.test.js]: Well, `jest` uses `spy.mockRestore()` in [their example](https://jestjs.io/docs/en/jest-object#jestspyonobject-methodname), so I guess we can leave it there.

**Comment by KubaJastrz** [comment]
I didn't make it in time with my revert commit :P
Gotta go back to work now, I will continue with tests in the evening.
